tensorflow/python/kernel_tests/embedding_ops_test.py

```diff
@@ -563,7 +563,12 @@ def testHigherRankMaxNorm(self):
           # Compare nonsharded to gather
           simple = embedding_ops.embedding_lookup(
               params, ids, max_norm=1.0).eval()
-          self.assertAllEqual(simple, array_ops.gather(params_norm, ids).eval())
+          # assertAllClose is used here as different implementations of sqrt are
+          # used to compute each of the values being compared.  These different
+          # implementations are not guaranteed to produce exactly the same
+          # results on all target platforms.  Therefore an exact comparison
+          # cannot be made.
+          self.assertAllClose(simple, array_ops.gather(params_norm, ids).eval())
```

Comment:
I'd probaby put details about the "AVX512 builds as the
embedding operation makes use of Eigen's fast vectorized square root
algorithm for doubles" etc.

Reply:
@protoget I've updated the comments to explicitly mention the AVX512 builds.

Reply:
@protoget Does the updated comment read okay to you now?  If not, I'm happy to change it.